scene/animation/tween.cpp

```diff
@@ -322,23 +322,16 @@ Variant &Tween::_get_initial_val(InterpolateData &p_data) {
 	return p_data.delta_val;
 }
 
-Variant &Tween::_get_delta_val(InterpolateData &p_data) {
-
-	// What kind of data are we interpolating?
+Variant &Tween::_get_final_val(InterpolateData &p_data) {
 	switch (p_data.type) {
-		case INTER_PROPERTY:
-		case INTER_METHOD:
-			// Simply return the given delta value
-			return p_data.delta_val;
-
 		case FOLLOW_PROPERTY:
 		case FOLLOW_METHOD: {
 			// We're following an object, so grab that instance
 			Object *target = ObjectDB::get_instance(p_data.target_id);
 			ERR_FAIL_COND_V(target == NULL, p_data.initial_val);
 
 			// We want to figure out the final value
-			Variant final_val;
+			static Variant final_val;
```

Comment:
This `static` worries me. Is it really necessary?

Reply:
statics removed